Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1776,6 +1776,22 @@ CanvasFingerprintingQuirkEnabled:
     WebCore:
       default: true
 
+CanvasHDREnabled:
```

Comment:
I would really love to avoid using the term HDR in WebCore as it quite vague, and have steered clear of it in the color work so far. 

For this setting, perhaps CanvasFloat16BackingStore or CanvasBackingStoreDataTypes would work?

Reply:
This patch here is only part 1 of N to implement https://github.com/w3c/ColorWeb-CG/blob/main/hdr_html_canvas_element.md , I was going to use this setting for the added colorspaces as well.

We've used "HDR" for other recent developments (HDRForImagesEnabled and WebGPUHDREnabled); it doesn't make it right, but that boat may have already sailed!

Would you have other suggestions for a setting related to "Adding support High Dynamic Range (HDR) imagery to HTML Canvas"?

Reply:
Should I move this new setting to a separate patch, to avoid confusing this feature-wide change from the small-part CanvasDataType addition?

Reply:
I think having two settings, one for the pixel formats and one for the color spaces is fine. 

Is this stuff really ready to be implemented? It doesn’t seem like a very complete spec, but perhaps I am misreading. 

Seems a little weird for canvas to have such a disjoint set of color spaces from CSS for instance.

Reply:
> Would you have other suggestions for a setting related to "Adding support High Dynamic Range (HDR) imagery to HTML Canvas"?

yeah, I would prefer those say what they were actually doing, which was probably support for specific color spaces/transfer encoding?

The fact that I can’t tell what it means is exactly the reason I am pushing to not use the term in WebKit.

Reply:
> Is this stuff really ready to be implemented? It doesn’t seem like a very complete spec, but perhaps I am misreading.
> 
> Seems a little weird for canvas to have such a disjoint set of color spaces from CSS for instance.

Reading this back, it sounds like I am pushing you not to work on this, which is not my intention. I am curious about why the spec is not in the usually W3C spec form, but this is sometimes how things go.

Reply:
Alright, I've renamed the setting `CanvasPixelFormatEnabled`, I can see the benefit now. Thank you for your review.

The feature is indeed lacking a proper spec, so it may still be unstable (as the setting denotes, and it's off by default). Hopefully ColorWeb-CG can work on it soon...
Also I've changed the names to match the current Chrome implementation (mainly dataType -> pixelFormat), to be able to compare the results from some early-adopter websites; this naming may still change of course.

Reply:
Feel free to comment at https://github.com/whatwg/html/issues/9461 if you're happy to share your opinions and suggestions there 😉